cmd/prometheus/main.go

```diff
@@ -198,6 +198,9 @@ func main() {
 	a.Flag("storage.tsdb.no-lockfile", "Do not create lockfile in data directory.").
 		Default("false").BoolVar(&cfg.tsdb.NoLockfile)
 
+	a.Flag("storage.tsdb.allow.overlapping.block", "Allow overlapping blocks which in-turn enables vertical compaction and vertical query merge.").
```

Comment:
I see dots are used to denoting a subsection and not to separate the words.
```suggestion
	a.Flag("storage.tsdb.allow-overlapping-blocks", "Allow overlapping blocks which in-turn enables vertical compaction and vertical query merge.").
```

Reply:
yeah I am also thinking it should be plural everywhere - AllowOverlappingBlock -> AllowOverlappingBlocks

Reply:
here is the change and I will update it here after it is merged
https://github.com/prometheus/tsdb/pull/537

Reply:
updated